Fixes #36656 - Pin @adobe/css-tools to 4.2.0

sjha4 opened the pull request:
<!---

Thank you for contributing to The Foreman, please read the
[following guide](https://www.theforeman.org/contribute.html), in short:

* [Create an issue](https://projects.theforeman.org/projects/foreman/issues)
* Reference the issue via `Fixes #1234` in the commit message
* Prefer present-tense, imperative-style commit messages
* Mark all strings for translation, see [1]
* Suggest prerequisites for testing and testing scenarios following example above.
* Prepend `[WIP]` for work in progress to prevent bots from triggering actions
* Be patient, we will do our best to take a look as soon as we can
* Explain the purpose of the PR, attach screenshots if applicable
* List all prerequisites for testing (e.g. VMware cluster, two smart proxies...)
* Reviewers often use extensive list of items to check, have a look prior submitting [2]
* Be nice and respectful

1: https://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers
2: https://github.com/theforeman/foreman/blob/develop/developer_docs/pr_review.asciidoc
-->

sjha4 committed:
Fixes #36656 - Pin @adobe/css-tools to 4.2.0

evgeni commented:
is that something that needs packaging, or shall we add it to the exclude list?

ofedoren commented:
Why exactly do we need to pin the lib instead of update node version in CI?..

jeremylenz commented:
> Why exactly do we need to pin the lib instead of update node version in CI?..

We're stuck on node 12 downstream for the time being, as I understand it. We had [previously upgraded node](https://github.com/Katello/katello/pull/10520) here in Github Actions and then had to revert it.

ofedoren commented:
> 
> We're stuck on node 12 downstream for the time being

oof :disappointed: Okay then. At least there is a reason...

sjha4 commented:
Updated to add this as dev-dependency and added it exclude list..I didn't see it being packaged and package-lock shows dev: true for this in foreman and katello.

jeremylenz reviewed (dismissed):
APJ :+1:

ekohl commented on package.json:
```diff
@@ -69,6 +69,7 @@
     "tabbable": "~5.2.0",
-    "sass": "~1.60.0"
+    "sass": "~1.60.0",
+    "@adobe/css-tools": "~4.2.0"
```
Nit: I prefer things to remain sorted. We already have `@` dependencies on top.

sjha4 replied:
Updated..
